client/app/telecom/dashboard/bills/telecom-dashboard-bills.controller.js

```diff
@@ -21,7 +21,8 @@ angular.module("managerApp").controller("TelecomDashboardBillsCtrl", function (O
 
     function getLastBills () {
         return OvhApiMeBill.Aapi().last().$promise.then(function (lastBills) {
-            self.last = lastBills.slice(Math.max(lastBills.length - self.amountBillsDisplayed, 0));
+            const lengtToDisplay = Math.max(lastBills.length - self.amountBillsDisplayed, 0);
+            self.last = _.sortBy(lastBills.slice(lengtToDisplay), "date").reverse();
```

Comment:
It's very strange to slice before sorting, because you have no idea what you're slicing. I know it's legacy but that's sad for so many other reasons.
Also, ```last``` is an horrible variable name.